C-Plus-Plus/cp/rain_water_trapping.cpp

```diff
@@ -0,0 +1,46 @@
+#include<bits/stdc++.h>
+using namespace std;
+#define li long int
+#define lli long long int
```

Comment:
Don't use CP tricks here

Reply:
I have done the required changes sir. Please check and review.